### PR TITLE
Various minor changes

### PR DESCRIPTION
**Pull request by joseph619**
Also removed Sy maintaining AZ bit, he doesn't maintain AZ anymore.

### CODE REVIEW

**Comment by jobobby04** [comment]
Well technically I do, just in a indirect way

**Comment by joseph619** [comment]
lol I meant direct. I could edit it again if you want.

**Comment by jobobby04** [comment]
I'm fine with it either way, though I wouldn't say I maintain it, just that I contribute to it

File: src/forks/README.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ This fork takes a new approach to the design of the base app along with several
 
 ## <img class="headerLogo" :src="$withBase('/forks/TachiyomiAZ/assets/logo.png')"> [TachiyomiAZ](/forks/TachiyomiAZ)
 ::: aside-guide
-Maintained by [az4521](https://github.com/az4521) and [Syer10](https://github.com/jobobby04)
+Maintained by [az4521](https://github.com/az4521)
 :::
 
-This hentai focused fork is based off of **[TachiyomiEH](/forks/TachiyomiEH)**, it has most features and fixes from **Tachiyomi**, as well as features from **[TachiyomiJ2K](/forks/TachiyomiJ2K)**. It has decided to keep the hamburger menu style of navigation, compared to the other forks that are moving to the new bottom navigation style.
+This hentai focused fork is based off of **[TachiyomiEH](/forks/TachiyomiEH)**, it has most features and fixes from **Tachiyomi**, as well as features from **[TachiyomiJ2K](/forks/TachiyomiJ2K)**. It has decided to keep the old hamburger menu style of navigation, compared to the other forks that have moved to the new bottom navigation style.
```
Comment (GHOSTBEAR) [inline src/forks/README.md:29]:
```suggestion
This hentai focused fork is based on **[TachiyomiEH](/forks/TachiyomiEH)**. It has most features and fixes from **Tachiyomi**, as well as features from **[TachiyomiJ2K](/forks/TachiyomiJ2K)**. It has decided to keep the old hamburger menu style of navigation, compared to the other forks that have moved to the new bottom navigation style.
```
Could you change `off of` to `on` and `, it` to `. It`. Just to make it a bit easier to read

**Comment by joseph619** [comment]
I actually removed it since it said maintained by az and Syer, but you don't maintain it any more right? Now it says

> Maintained by [az4521](https://github.com/az4521)

File: src/forks/README.md [hunk 2]
```diff
@@ -41,4 +41,4 @@ Maintained by [NerdNumber9](https://github.com/NerdNumber9)
 :::
 This fork adds features that enhances the hentai experience with integrations to popular sevices such as **E-Hentai**, **nHentai** and more.
 
-At the moment, it is **deprecated** as the developer does not have time to maintain it. Use [TachiyomiAZ](/forks/TachiyomiAZ) instead.
+At the moment, it is **deprecated** as the developer does not have time to maintain it. Use [TachiyomiAZ](/forks/TachiyomiAZ) or  [TachiyomiSY](/forks/TachiyomiSY) instead.
```
Comment (GHOSTBEAR) [inline src/forks/README.md:44]:
```suggestion
At the moment, it is **deprecated** as the developer does not have time to maintain it. Use [TachiyomiAZ](/forks/TachiyomiAZ) or [TachiyomiSY](/forks/TachiyomiSY) instead.
```
Remove the extra space between `or` and `[TachiyomiSY](/forks/TachiyomiSY)`

Comment (joseph619) [reply src/forks/README.md]: Done

File: src/forks/README.md (hunk 1 above)
Comment (joseph619) [reply src/forks/README.md]: Done

**Pull request merged**